CR2WTests/StressTest.cs

```diff
@@ -58,7 +58,7 @@ public class StressTest
         [ClassInitialize]
         public static void Setup(TestContext context)
         {
-            var exedir = LookUpW3exe();
+            var exedir = new FileInfo(LookUpW3exe()).Directory.FullName;

```

Comment:
Can we make an env var like "W3_DIR" and check if thats avaliable, otherwise w3lookup otherwise error?

Reply:
Yes, we should do that, but the unit test is broken right now anyways because max changed some things in the way memorymapped bundles are read...

Reply:
on it, my bad